src/core/client/ui/shared/typography.css

```diff
@@ -127,11 +127,29 @@
   letter-spacing: calc(0.2em / 16);
 }
 
-.textField {
+.validationMessage {
   color: var(--palette-common-black);
-  font-family: "Source Sans Pro";
+  font-family: var(--font-family);
+  font-weight: var(--font-weight-medium);
+  font-size: calc(16rem / var(--rem-base));
+  line-height: normal;
```

Comment:
Use line height from figma designs, should be `calc(16rem / var(--rem-base));`

Reply:
Why is it different from normal? - And Figma says `normal`. Do you have another config? I'm checking the CSS tab